packages/rest/src/controllers/credentials/CredentialController.ts

```diff
@@ -31,8 +31,17 @@ export class CredentialController extends Controller {
    */
   @Example<CredentialExchangeRecordProps[]>([CredentialExchangeRecordExample])
   @Get('/')
-  public async getAllCredentials() {
-    const credentials = await this.agent.credentials.getAll()
+  public async getAllCredentials(
+    @Query('threadId') threadId?: string,
+    @Query('connectionId') connectionId?: string,
+    @Query('state') state?: string,
```

Comment:
Does it work if we use `CredentialState` as the type here? Maybe that'll give better validation and documentation in the swagger ui?

Reply:
<img width="634" alt="image" src="https://user-images.githubusercontent.com/23165168/189352496-f53e5440-a8b0-4b57-80ba-e0da5f68a0ab.png">